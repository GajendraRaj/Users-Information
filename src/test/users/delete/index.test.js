import React from "react";
import { shallow } from "enzyme";
import DeleteUser from "../../../components/users/delete";

describe("DeleteUser component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<DeleteUser />);
  });

  it("should render component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render Delete button delete the record", () => {
    expect(wrapper.find("button").at(0).text()).toEqual("Delete");
  });

  it("should render model popup on Delete button click", () => {
    const deleteButton = wrapper.find("button").at(0);
    deleteButton.simulate("click");
    expect(wrapper.find("span").at(0).text()).toEqual("Delete User");
  });
});
