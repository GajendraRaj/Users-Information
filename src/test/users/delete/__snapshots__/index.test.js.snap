// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteUser component should render component correctly 1`] = `ShallowWrapper {}`;
